task/orka-full/0.1/SCRIPTS.md

```diff
@@ -0,0 +1,121 @@
+# Install & Configure with Scripts
+
+For easier install and configuration of the Orka Tekton `Tasks`, you can use the provided scripts.
+
+* [TL;DR;](#tldr)
+* [How to install the task](#how-to-instal-the-task)
+* [How to store your credentials](#how-to-store-your-credentials)
+
+## TL;DR;
+
+Omit `NAMESPACE` if installing in the `default` Kubernetes namespace.
+
+#### Install the task
```

Comment:
`nit`: suggest changing these to size 3 headings here, e.g. `### Install the task`

Reply:
The H4 is deliberate. H3 just doesn't look good (it's too big and becomes a bit redundant with the more detailed section of the same name below). I was looking for a way to implement sectioning but still retain anchors. Otherwise, I would have gone with plain bold formatting.

Let me know what you think.

Reply:
Makes sense :)